api/android/build-android-lib.sh

```diff
@@ -87,6 +90,12 @@ enable_nnfw="yes"
 # Enable SNPE
 enable_snpe="no"
 
+# Enable PyTorch
+enable_pytorch="no"
+
+#Set PyTorch version (available: 1.8.0 (unstable))
```

Comment:
Minor comment: need a space between '#' and 'S'.

Reply:
Thanks!